src/Microsoft.AspNet.Server.Kestrel/Infrastructure/Constants.cs

```diff
@@ -20,5 +23,18 @@ internal class Constants
         /// for info on the format.
         /// </summary>
         public const string RFC1123DateFormat = "r";
+
+        public readonly static int VectorSpan = Vector<byte>.Count;
```

Comment:
`Vector<byte>.Count` should Jit to const when intrinsic; not sure when `Vector.IsHardwareAccelerated == false`; as the backing static currently isn't `readonly` (https://github.com/dotnet/corefx/issues/5152) however we know `readonly static int` does Jit to const; otherwise so intermediate `readonly static` for that.